theme/themes/pastanaga/extras/blocks.less

```diff
@@ -783,6 +775,14 @@ body.has-toolbar.has-sidebar-collapsed .ui.wrapper > .ui.inner.block.full {
     // reseting the default Accordion behavior
     display: initial;
   }
+  .ui.form .ui.input input {
+    //reset for semantic-ui outline:none
+    border-width: @1px;
+    border-color: transparent;
+    &:focus {
+      border-color: @blueFocus;
```

Comment:
@Wagner3UB @sneridagh for the focus of the input I would use
@focusedFormBorderColor instead of @blueFocus
https://github.com/plone/volto/blob/main/theme/themes/default/globals/site.variables#L96
this is because Volto uses this rule for the sidebar textarea and inputs
so it would make sense to keep it consistent.
![focus-of-inputs](https://github.com/plone/volto/assets/152852/8d85e05c-1252-4369-be64-0b6f71aa5ce2)